ARGO/include/AiSystem.h

```diff
@@ -7,14 +7,15 @@
 #include "AiComponent.h"
 #include "TransformComponent.h"
 #include "ForceComponent.h"
+#include "HealthComponent.h"
 #include "AiStates.h"
 #include "Utilities.h"
 #include "BehaviourTree.h"
 
 class AiSystem : public BaseSystem
 {
 public:
-	AiSystem(Entity(&t_players)[Utilities::S_MAX_PLAYERS], std::vector<Entity>& t_enemies, EventManager& t_eventManager);
+	AiSystem(Entity(&t_players)[Utilities::S_MAX_PLAYERS], Entity(&t_enemies)[Utilities::ENEMY_POOL_SIZE] , EventManager& t_eventManager);
```

Comment:
have a comment here please as this is a bit of a cluster fudge

Reply:
This is the proper syntax for passing in a reference to an array of a specific size. Does it need a comment?